nemo_curator/modules/semantic_dedup/semanticclusterleveldedup.py

```diff
@@ -51,31 +52,52 @@ def __init__(
 
         Args:
             n_clusters (int): Number of clusters.
-            emb_by_clust_dir (str): Directory containing embeddings by cluster.
-            sorted_clusters_dir (str): Directory containing sorted clusters.
```

Comment:
We deprecate `emb_by_clust_dir` and `sorted_clusters_dir` in favor of `cache_dir` and `embeddings_save_loc` for `SemanticClusterLevelDedup`, which matches the logic in the `SemDedup` class.